client/src/components/PortfolioFilters.tsx

```diff
@@ -0,0 +1,394 @@
+import { I18n } from "@lingui/core";
+import { t, Trans, Plural } from "@lingui/macro";
+import classnames from "classnames";
+import React from "react";
+import { CheckIcon, ChevronIcon, InfoIcon } from "./Icons";
+import { Multiselect } from "./Multiselect";
+import { FilterContext, FilterNumberRange, MINMAX_DEFAULT } from "./PropertiesList";
+import "styles/PortfolioFilters.scss";
+import FocusTrap from "focus-trap-react";
+import { Alert } from "./Alert";
+import Modal from "./Modal";
+import { LocaleLink } from "i18n";
+import { createWhoOwnsWhatRoutePaths } from "routes";
+import { isLegacyPath } from "./WowzaToggle";
+import { useLocation } from "react-router-dom";
+
+type PortfolioFiltersProps = {
+  i18n: I18n;
+};
+export const PortfolioFilters = React.memo(
+  React.forwardRef<HTMLDivElement, PortfolioFiltersProps>((props, ref) => {
+    const { i18n } = props;
+
+    const [showInfoModal, setShowInfoModal] = React.useState(false);
+    const [showLandlordModal, setShowLandlordModal] = React.useState(false);
+    const { pathname } = useLocation();
+    const { about, methodology, legacy } = createWhoOwnsWhatRoutePaths();
+
+    const { filterContext, setFilterContext } = React.useContext(FilterContext);
+    const { filteredBuildings } = filterContext;
+    const { ownernames: ownernamesOptions, zip: zipOptions } = filterContext.filterOptions;
+    const { ownernames: ownernamesSelections, zip: zipSelections } = filterContext.filterSelections;
+
+    const [rsunitslatestActive, setRsunitslatestActive] = React.useState(false);
+    const updateRsunitslatest = () => {
+      setRsunitslatestActive(!rsunitslatestActive);
```

Comment:
noting that we need telemetry/logging for all of these, but no worries if that's already scheduled for a future PR

Reply:
Thanks for catching this! Yeah, I'll add all of the telemetry in a separate PR, and listing all the ones you've pointed out.